<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QWidget" name="Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="grassView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>79</width>
     <height>74</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="grassLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="dirtView">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>79</width>
     <height>74</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="dirtLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="stoneView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>79</width>
     <height>74</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="waterLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="stoneLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="waterView">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>79</width>
     <height>74</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lavaLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lavaView">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>79</width>
     <height>74</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="sandView">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>79</width>
     <height>74</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="snowLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="snowView">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>79</width>
     <height>74</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="sandLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>lavaView</zorder>
  <zorder>snowView</zorder>
  <zorder>waterView</zorder>
  <zorder>grassView</zorder>
  <zorder>grassLabel</zorder>
  <zorder>dirtView</zorder>
  <zorder>dirtLabel</zorder>
  <zorder>stoneView</zorder>
  <zorder>waterLabel</zorder>
  <zorder>stoneLabel</zorder>
  <zorder>lavaLabel</zorder>
  <zorder>sandView</zorder>
  <zorder>snowLabel</zorder>
  <zorder>sandLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
